grade_test.go

```diff
@@ -331,7 +358,7 @@ func TestParseInFileWithInput(t *testing.T) {
 
 	actual := parseInFile(tmp.Name())
 	expected := []string{"test", "multiline ", "input", "for", "program", ""}
-
+	defer os.Remove(tmp.Name())
```

Comment:
duplicate

Reply:
these should go directly below os.CreateTemp calls for clarity

Reply:
That makes sense, I"m going to fix.